From 527f6e7bd984289a8eda22eb0a0f33a99a36636f Mon Sep 17 00:00:00 2001
From: Bradley Lowekamp <blowekamp@mail.nih.gov>
Date: Wed, 13 Apr 2022 09:27:39 -0400
Subject: [PATCH] WIP: Disable server side search

---
 Utilities/Doxygen/DoxygenConfig.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Utilities/Doxygen/DoxygenConfig.cmake b/Utilities/Doxygen/DoxygenConfig.cmake
index bda19553ba..aae4d5deaf 100644
--- a/Utilities/Doxygen/DoxygenConfig.cmake
+++ b/Utilities/Doxygen/DoxygenConfig.cmake
@@ -74,7 +74,7 @@ set(DOXYGEN_ECLIPSE_DOC_ID "org.itk.ITK")
 set(DOXYGEN_ENUM_VALUES_PER_LINE "1")
 set(DOXYGEN_USE_MATHJAX "YES")
 set(DOXYGEN_MATHJAX_RELPATH "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.7/")
-set(DOXYGEN_SERVER_BASED_SEARCH "YES")
+set(DOXYGEN_SERVER_BASED_SEARCH "NO")
 set(DOXYGEN_LATEX_CMD_NAME "latex")
 set(DOXYGEN_PAPER_TYPE "letter")
 set(DOXYGEN_EXTRA_PACKAGES "amsmath" "amsfonts")
-- 
2.24.3 (Apple Git-128)

